packages/react/src/__tests__/FormControl.test.tsx

```diff
@@ -390,6 +391,41 @@ describe('FormControl', () => {
         expect(consoleSpy).toHaveBeenCalled()
         consoleSpy.mockRestore()
       })
+
+      it('should not add required prop to individual checkbox', async () => {
```

Comment:
>In other cases when signing up on a site, there may be some options like marketing emails and terms and conditions, where terms and conditions is required but marketing emails is not.

It makes me think what if the terms and conditions option is required (which is in most cases) and there are no other options. Does that mean individual checkboxes can also be required even if there are not a part of group? 

Let me know if I am missing something here.

Reply:
That's a good point, @ichelsea thoughts on this?

Reply:
No technical limitations, just wanted to confirm since that wasn't previously allowed. thanks for the input!